packages/std/src/results/cosmos_msg.rs

```diff
@@ -19,6 +19,15 @@ where
     // to call into more app-specific code (whatever they define)
     Custom(T),
     Staking(StakingMsg),
+    /// A Stargate message encoded the same way as a protobof [Any](https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/any.proto).
+    /// This is the same structure as messages in `TxBody` from [ADR-020](https://github.com/cosmos/cosmos-sdk/blob/master/docs/architecture/adr-020-protobuf-transaction-encoding.md)
+    #[cfg(feature = "stargate")]
+    Stargate {
+        /// 'type' is a reserved keyword, so we need to use a serde rename
+        #[serde(rename = "type")]
+        kind: String,
```

Comment:
You can use a reserved keyword by prefixing it with `r#`, e.g. `r#type: String,` (https://doc.rust-lang.org/edition-guide/rust-2018/module-system/raw-identifiers.html).

However, in this case let's just use the field names from `Any`: `type_url`/`value` (https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/any.proto#L154-L157)